packages/eslint-plugin/tests/rules/space-infix-ops.test.ts

```diff
@@ -977,6 +977,26 @@ ruleTester.run('space-infix-ops', rule, {
         },
       ],
     },
+    {
+      code: `
+        type Test<T> = T extends boolean?true:false
```

Comment:
A few edge cases to test for...

* What if there's only a space before, not after? And vice-versa?
* Placing newlines at various points in the ternary, with and without spaces